decompose/src/commonMain/kotlin/com/arkivanov/decompose/router/RouterExt.kt

```diff
@@ -32,9 +32,19 @@ fun <C : Any> Router<C, *>.pop(onComplete: (isSuccess: Boolean) -> Unit = {}) {
 
 /**
  * Drops the configurations at the top of the stack while the [predicate] returns true
+ *
+ * @param onComplete called when the navigation is finished (either synchronously or asynchronously).
+ * The `isSuccess` argument is `true` if the stack size was greater than 1 and a component was popped,
```

Comment:
This statement is not quite correct for this case. Would it be possible to change this to something like the following?

> The `isSuccess` argument is `true` if at least one component has been popped.